python/tests/test_fastexcel.py

```diff
@@ -0,0 +1,80 @@
+from os.path import dirname
```

Comment:
Is it possible to also have tests for the old style Excel format ?
You can have as fixtures those : 
- https://github.com/ToucanToco/peakina/blob/main/tests/fixtures/0_2.xls
- https://github.com/ToucanToco/peakina/blob/main/tests/fixtures/fixture-skip-rows.xls
- https://github.com/ToucanToco/peakina/blob/main/tests/fixtures/fixture_new_format.xls

Reply:
Excellent remark @Sanix-Darker ! But I figured it would be done in another PR.
Clearly we need to add many excel files (with formulas, with dates, with header on row 3, with different formats like xlsx, xls, xlsb...)

Reply:
Sure! But I mainly focused on excel for now, so I planned on adding `.xls` and `.odf` later on